### ISSUE TITLE
"Chunk build failed"

### ISSUE THREAD

**Issue by danRP408** [GTNewHorizons/Angelica#591]
### Your GTNH Discord Username

danRP#6881

### Mod Version

1.0.0-beta4

### Java Version

Java 8

### Graphics Card Vendor

NVidia

### Bug Report

game crashes when entering a world
crash log: https://mclo.gs/ktu8xYn
fml latest log: https://mclo.gs/NaxvQGH

### Mod List or GTNH Pack Version

- unimixins 0.1.17
- angelica 1.0.0-beta4
- gtnhlib 0.4.0

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)
- [X] This issue is not related to a feature that is disabed by default - Shaders, MCPF, etc.  [They'll be enabled when they're ready for testing]

**Comment by Caedis** [comment]
please post fml-client-latest.log. Should be much longer

**Comment by danRP408** [comment]
https://mclo.gs/O67mqdg

**Comment by Caedis** [comment]
@mitchej123
https://github.com/GTNewHorizons/GTNHLib/blob/master/src/main/java/com/gtnewhorizon/gtnhlib/core/GTNHLibCore.java#L26 needs updating

**Comment by Caedis** [comment]
if i had to guess, it should be `com.gtnewhorizon.gtnhlib.core.transformer.TessellatorRedirectorTransformer`

**Comment by mitchej123** [comment]
Yes I renamed it after testing it. 🤦    Should have used class.getName...
